package.json

```diff
@@ -8,6 +8,7 @@
   "dependencies": {
     "remark-cli": "^7.0.0",
     "remark-frontmatter": "^2.0.0",
+    "remark-lint-no-dead-urls": "^1.1.0",
```

Comment:
this should not be part of this commit, but promising 👏

Reply:
Mmh I added it since it was useful to detect dead urls, but I do not want to run it in the CI as it takes a bit of time.